core/modules/system/config/system.core.json

```
@@ -87,6 +87,7 @@
     "user_mail_status_canceled_notify": 0,
     "user_register": "admin_only",
     "user_signatures": 0,
+    "user_page_tabs": false,
```

Lets explicitly add the word `login` to this setting, to make it less vague/ambiguous:
```suggestion
    "user_login_page_tabs": false,
```